### DISCUSSION TITLE
✨ [Feature]: 关于导入第三方组件问题

### DISCUSSION THREAD

**Discussion by lhdpfc** [opentiny/tiny-engine#131] [Q&A]
### What problem does this feature solve

关于导入第三方组件问题：
1.关于如何导入第三方组件不使用npm.script方式导入，比如 
  "package": "@huawei/tiny-vue3", // npm 包名
    "version": "0.1.16", // 版本号
    "destructuring": true, // 是否解构
    "exportName": "Button" // npm package 导出名

我想用这种方式导用第三方比如element-plus或antd vue 有没有文档说明？

### What does the proposed API look like

关于导入第三方组件问题：
1.关于如何导入第三方组件不使用npm.script方式导入，比如 
  "package": "@huawei/tiny-vue3", // npm 包名
    "version": "0.1.16", // 版本号
    "destructuring": true, // 是否解构
    "exportName": "Button" // npm package 导出名

我想用这种方式导用第三方比如element-plus或antd vue 有没有文档说明？

[a comment by an automated account is not shown]

**Comment by chilingling** [comment]
非 bug ，转为 discussions

[a comment by an automated account is not shown]

**Comment by chilingling** [comment]
官方文档：https://opentiny.design/tiny-engine#/help-center/course/engine/15
有问题可以继续讨论
